src/components/molecules/CampaignsGrid/index.tsx

```
@@ -0,0 +1,171 @@
+                  </Link>
+                </Button>
+              )}
+            </div>
+            <CardDescription className="text-brand-200">
+              {campaign.description}
+            </CardDescription>
+          </CardHeader>
+          <CardContent className="space-y-4">
+            {campaign.players &&
+              campaign.sessions &&
+              campaign.players[0].hasOwnProperty("count") &&
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)